cli/cmd/util.go

```diff
@@ -58,7 +61,11 @@ func verifyCoordinator(host string, configFilename string, insecure bool) ([]*pe
 
 	// get certificate using provided config
 	if configFilename != "" {
-		pemBlock, _, err := era.GetCertificate(host, configFilename)
+		pemBlock, tcbLevel, err := era.GetCertificate(host, configFilename)
+		if err == attestation.ErrTCBLevelInvalid && util.StringSliceContains(acceptedTcbLevels, tcbLevel.String()) {
```

Comment:
```suggestion
		if errors.Is(err, attestation.ErrTCBLevelInvalid) && util.StringSliceContains(acceptedTcbLevels, tcbLevel.String()) {
```
A bit cleaner, since this will also return true if the error contains a wrapped `ErrTCBLevelInvalid` error